do-not-commit/exceptions.py

```diff
@@ -0,0 +1,24 @@
+# ruleid:raise-not-base-exception
+raise "error here"
+
+# todoruleid:raise-not-base-exception
+raise 5
```

Comment:
In Python3, a runtime `TypeError` will be thrown if you attempt to raise an object or class which does not inherit from `BaseException` ([.exceptions.exceptions.raise-not-base-exception](https://semgrep.dev/r/.exceptions.exceptions.raise-not-base-exception))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=.exceptions.exceptions.raise-not-base-exception) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>